src/test/cypress/support/pageobjects/ExamStartEndPage.ts

```diff
@@ -5,15 +5,27 @@ export class ExamStartEndPage {
         artemis.users.getAccountInfo((account: any) => cy.get('#fullname').type((account.firstName ?? '') + ' ' + (account.lastName ?? '')));
     }
 
-    setConfirmCheckmark() {
-        cy.get('#confirmBox').click();
+    setConfirmCheckmark(timeout?: number) {
+        cy.get('#confirmBox', { timeout }).click();
     }
 
-    startExam() {
+    pressStart() {
         cy.contains('Start').click();
     }
 
-    finishExam() {
+    pressFinish() {
         cy.get('.btn').contains('Finish').click();
     }
+
+    startExam() {
+        this.setConfirmCheckmark();
+        this.enterFirstnameLastname();
+        this.pressStart();
+    }
+
+    finishExam(timeout = 20000) {
```

Comment:
this is intentionally not the case because i need this check to wait for a minimum of 20 seconds. If the env changes this would break. I do agree that it should be done in the test though.